drivers/flash/nrf_qspi_nor.c

```diff
@@ -1293,8 +1321,15 @@ static int enter_dpd(const struct device *const dev)
 static int exit_dpd(const struct device *const dev)
 {
 	if (IS_ENABLED(DT_INST_PROP(0, has_dpd))) {
+		/* Buffers for dummy data from RDPD */
+		uint8_t rx_buf[4] = { 0 };
+		struct qspi_buf rx = {
+		       .buf = rx_buf,
+		       .len = 4
+		};
 		struct qspi_cmd cmd = {
-			.op_code = SPI_NOR_CMD_RDPD,
+		       .op_code = SPI_NOR_CMD_RDPD,
+		       .rx_buf = &rx
```

Comment:
What's the point in reading these dummy bytes?

Reply:
Our flash device (W25Q128JV) didn't reliably come out of DPD unless we read out the ID bytes it returned as the result of RDPD

Reply:
Is this W25Q128JV specific or happens with other vendors?
If this does not happen with other vendors shouldn't this be marked as vendor/device specific quirk?

Reply:
After going through the datasheets, and retrying, looks like this issue was maybe caused by something else, and it's working properly without the rx buffer here, I'll pull that out.